### DISCUSSION TITLE
Discover's Update and Refresh icons fail to appear in dark mode.

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#84] [General]
https://snipboard.io/icFzpw.jpg

Fortunately, hover still works!

**Comment by geckolinux** [comment]
Hi there, thanks for reporting. Sounds like an upstream bug. You might want to play around a bit more with your custom theme colors and/or custom icon theme. But that's one of the reasons why SpiralLinux only officially supports and recommends the default theme. If a SpiralLinux edition includes a dark theme it's only the result of it being automatically installed as a dependency, but it's not tested or recommended.

**Comment by dacomboman** [comment]
True.
Just installed a recently added theme Catalina-Dark, and it works, fine and dandy. ;)
